docs/reference/versions/compliant.md

```diff
@@ -0,0 +1,24 @@
+# Compliant Cloud
```

Comment:
I am not into explaining which version of openstack cloud / ceph we deliver. This is an operational impact.
I would prefer if we were to expose the minimum API version. This is what ppl need to know.

Reply:
You can ignore my comment above as of now, as the same information is published on our main site.

I am not fond of duplicating the information, as now we have two places to edit. 

I encourage you to remove it once again. Yet, the content itself is not hurting, as of today.